tests-e2e/cypress/integration/zoom_setup_wizard.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// <reference path="../support/index.d.ts" />
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const defaultZoomConfig = {
+    "accountlevelapp": false,
+    "apikey": "",
+    "apisecret": "",
+    "enableoauth": true,
+    "encryptionkey": "",
+    "oauthclientid": "",
+    "oauthclientsecret": "",
+    "webhooksecret": "",
+    "zoomapiurl": "",
+    "zoomurl": ""
+}
+
+describe('Zoom setup wizard', () => {
+    beforeEach(() => {
+        // # Login as sysadmin
+        cy.apiAdminLogin();
+
+        cy.apiDeleteAllPostsInDirectChannel('sysadmin', 'zoom');
+
+        cy.apiUpdateConfig({
+            PluginSettings: {
+                Plugins: {
+                    zoom: defaultZoomConfig,
+                },
+            },
+        });
+
+        cy.apiDisablePluginById('zoom');
+        cy.apiEnablePluginById('zoom');
+
+        cy.visit('/test/messages/@zoom');
+    });
+
+    it.only('Zoom setup flow', () => {
+        cy.apiGetConfig().then(({config}) => {
+            const zoomSettings = config.PluginSettings.Plugins.zoom;
+            const withGenerated = {
+                ...defaultZoomConfig,
+                encryptionkey: zoomSettings.encryptionkey,
+                webhooksecret: zoomSettings.webhooksecret,
+            }
+
+            expect(zoomSettings).to.deep.equal(withGenerated);
+        });
+
+        cy.get('#post_textbox').clear().type('/zoom setup');
+        cy.get('#post_textbox').type(' ');
+        cy.get('#post_textbox').type('{enter}');
+
+        let buttons = [
+            ['Continue', 'Welcome to Zoom for Mattermost!'],
+            ['No', 'Are you using a self-hosted private cloud or on-prem Zoom server?'],
+            ['Continue', 'Go to https://marketplace.zoom.us'],
+            ['Continue', 'Choose Account-level app as the app type.'],
+            ['Enter Client ID and Client secret', 'In the App Credentials tab, note the values for Client ID and Client secret'],
+        ]
+
+        buttons.forEach(handleClickStep);
+
+        // Enter credentials into interactive dialog
```

Comment:
You almost use no comments to explain what is going on, but in this particular comment, you are not using the "standard" that is stated at the header of the file.